pf/internal/pfutils/proposed_new_test.go

```diff
@@ -28,12 +30,16 @@ import (
 )
 
 type testcase struct {
-	Schema Schema
+	Schema runtypes.Schema
 	Prior  valueBuilder
 	Config valueBuilder
 	Want   valueBuilder
 }
 
+type runtiemSchemaAdapter struct{ Schema }
```

Comment:
```suggestion
type runtimeSchemaAdapter struct{ Schema }
```

typo in the whole file